src/main/resources/changesets/changelog-consent-5.0.xml

```diff
@@ -5,20 +5,22 @@
          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <changeSet author="ssaucedo" id="30">
+        <validCheckSum>ANY</validCheckSum>
         <addColumn tableName="consents">
             <column name="name" type="varchar(255)">
                 <constraints nullable="false"/>
             </column>
-            <column name="structuredDataUseLetter" type="text">
+            <column name="structureddatauseletter" type="text">
                 <constraints nullable="true"/>
             </column>
         </addColumn>
     </changeSet>
 
 
     <changeSet author="ssaucedo" id="31">
-        <sql dbms="hsql, mysql">
-            update consents t1, consents t2 set t1.name = t2.consentId where t1.consentId = t2.consentId
+        <validCheckSum>ANY</validCheckSum>
+        <sql dbms="mysql">
```

Comment:
again dbms="mysql"

Reply:
Could have also been `dbms="hsqldb, mysql"`, but yes, to restrict them from failing in postgres.